WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -3850,4 +3851,411 @@ public void testAssignmentApi() throws Exception
+
+        // Creates test data.
+        myReusableFunctions.insertValueIntoNecropsyDataset("necropsyTestId1", new Date());
+
+        // Runs test email in the browser.
+        EHRAdminPage.beginAt(this, "/ehr/" + getContainerPath());
+        EHRAdminPage.beginAt(this, "/ehr/" + getContainerPath()).clickNotificationService(this);
+        waitAndClickAndWait(Locator.tagWithAttributeContaining("a", "href", "wnprc_ehr.notification.DeathNotificationRevamp").withText("Run Report In Browser"));
+
+        // Validates necessary info.
+        assertTextPresent("necropsyTestId1 has been marked as dead.");
+
+        // Finishes test.
+        log("Completed notification revamp test: Death Notification Revamp");
+    }
+
+    public void notificationRevampTestPrenatalDeathNotificationRevamp() throws UnhandledAlertException, IOException, CommandException {
+        // Setup
+        log("Starting notification revamp test: Prenatal Death Notification Revamp");
+        ReusableTestFunctions myReusableFunctions = new ReusableTestFunctions();
+
+        // Navigates to home to get a fresh start.
+        myReusableFunctions.goHome();
+
+        // Creates test data.
+        myReusableFunctions.insertValueIntoNecropsyDataset("pdNecropsyTestId2", new Date());
+
+        // Runs test email in the browser.
+        EHRAdminPage.beginAt(this, "/ehr/" + getContainerPath());
+        EHRAdminPage.beginAt(this, "/ehr/" + getContainerPath()).clickNotificationService(this);
+        waitAndClickAndWait(Locator.tagWithAttributeContaining("a", "href", "wnprc_ehr.notification.DeathNotificationRevamp").withText("Run Report In Browser"));
+
+        // Validates necessary info.
+        assertTextPresent("pdNecropsyTestId2 has been marked as dead.");
+        assertTextPresent("Dam");
+
+        // Finishes test.
+        log("Completed notification revamp test: Prenatal Death Notification Revamp");
+    }
+
+    @Test
+    public void notificationRevampSetup() throws UnhandledAlertException, IOException, CommandException {
+        // Set up.
+        log("Starting notificationRevampSetup()");
+        ReusableTestFunctions myReusableFunctions = new ReusableTestFunctions();
+
+        // Navigates to home to get a fresh start.
+        myReusableFunctions.goHome();
+
+        // Navigates to admin notifications page.
+        EHRAdminPage ehrAdminPage = EHRAdminPage.beginAt(this, "/ehr/" + getContainerPath());
+        NotificationAdminPage notificationAdminPage = ehrAdminPage.clickNotificationService(this);
+        // Updates the notification user and reply email.
+        notificationAdminPage.setNotificationUserAndReplyEmail(DATA_ADMIN_USER);
+        // Enables all notification that we will be testing. //TODO a34: Can we remake a function with an appropriate name for this?  Something like "enableNotification" maybe.
+//        notificationAdminPage.enableDeathNotification("status_org.labkey.wnprc_ehr.notification.BloodDrawsTodayAll");
```

Comment:
why are all these lines commented?

Reply:
Commented out because I'm currently not using Dumbster, but if any future notifications use Dumbster I'll need to use this code to reenable them.  Keeping it here temporarily for reference.